src/profile-logic/gecko-profile-versioning.js

```diff
@@ -331,5 +331,78 @@ const _upgraders = {
     }
     convertToVersionTenRecursive(profile);
   },
+  [11]: profile => {
+    const categories = [
+      {
+        name: 'Idle',
+        color: 'transparent',
+      },
+      {
+        name: 'Other',
+        color: 'grey',
+      },
+      {
+        name: 'JavaScript',
+        color: 'yellow',
+      },
+      {
+        name: 'Layout',
+        color: 'purple',
+      },
+      {
+        name: 'Graphics',
+        color: 'green',
+      },
+      {
+        name: 'DOM',
+        color: 'blue',
+      },
+      {
+        name: 'GC / CC',
+        color: 'orange',
+      },
+      {
+        name: 'Network',
+        color: 'lightblue',
+      },
+      {
+        name: 'HTML Parsing',
+        color: 'brown',
+      },
+    ];
```

Comment:
I discover this new format. I wonder if we shouldn't have it more similar to other tables: have a `categoryTable` property in the first level of the profile object (not in `meta`), with 2 arrays `name` and `color` and a `length` property.

Trying to challenge your choices to see if they're well thought ;)

Reply:
The `schema` + `data` format is appropriate for tables that have lots and lots of entries, because it avoids repeating the column names in the JSON. It's also useful if some of the columns are null on many entries, because those columns can be ordered at the end, so that an entry can set its values for these columns to null implicitly just by truncating its data array.

These properties are not needed for the categories list: This list only has two columns (neither of which will be null) and only around 10 entries. I chose the more natural JSON representation for it because it's easier to read and easier to work with. It's also consistent with the format of the `libs` array.

Maybe we should make the `extensions` list use the more natural format, too.

Reply:
Oh, I don't have an opinion on whether this table should be inside `meta` or outside of it. The `extensions` list is inside of it so it felt like the right place.

Reply:
But the `libs` list is outside of it. I really have no opinion either way.